typescript/web/src/components/website/Hero/Hero.tsx

```diff
@@ -128,27 +120,21 @@ export function Hero() {
               </Link>
             </Text> */}
           </Box>
-          <Box
+          <Flex
+            direction="column"
+            justify="center"
+            flexGrow={1}
             pos="relative"
-            w={{ base: "full", lg: "560px" }}
+            MaxH="0"
```

Comment:
The property is incorrectly named. It should be `maxH` instead of `MaxH`.
Please fix it and verify that it doesn't break things.

Reply:
OK